### DISCUSSION TITLE
UI redesign suggestions for main page

### DISCUSSION THREAD

**Discussion by pkrasicki** [LemmyNet/lemmy-ui#1503] [Ideas]
Hi, I would like to suggest a few UI improvements for the main page and the light theme. I think you could make the page easier to read by doing only a few small changes.

**Before**
![before](https://github.com/LemmyNet/lemmy-ui/assets/49721209/2dd457bb-03ea-46e7-a451-a5d28a817b3b)

**After**
![after](https://github.com/LemmyNet/lemmy-ui/assets/49721209/c8d90d1d-4e6a-45bf-90f4-b2813a25c8b4)

**Changes:**
- changed the background color of filters to a less distracting dark gray color `rgb(73, 80, 87)`, which was already used for text
- removed the background and border for the cards on the right side
- the green buttons on the right no longer have a background to make them less distracting. I made their border and text a darker shade of green `#008e3b` to make the text easier to read
- the color of orange links is now darker `#ce4500` to also make them easier to read. The original colors didn't have enough contrast, which you can see in Firefox (or Chrome) accessibility tool

Original contrast for green text:
![contrast1](https://github.com/LemmyNet/lemmy-ui/assets/49721209/51f04c16-6684-4cf7-8ee3-54c297d82661)

Original contrast for orange text:
![contrast2](https://github.com/LemmyNet/lemmy-ui/assets/49721209/2271f914-4b27-48ae-9616-e981ca027c08)

**Comment by krestenlaust** [comment]
Have you considered the [principles of gestalt](https://www.usertesting.com/resources/topics/gestalt-principles) in your redesign?

It seems to me that this change would make it harder to distinct between the columns, which in turn would make the overall page more cluttered.

Suggested rename: "UI redesign suggestions for main page"

**Comment by pkrasicki** [comment]
If you are saying that we should make the page background darker and other backgrounds white, then yeah that's also a good idea. Here I set the body background color to the previously used card background color `#f8f9fa`.

![bg2-1](https://github.com/LemmyNet/lemmy-ui/assets/49721209/00da629d-c322-45db-9986-7bff71e463db)

This might be too subtle, so here is an example with a bit darker body color `#f4f4f4`.
![bg2-2](https://github.com/LemmyNet/lemmy-ui/assets/49721209/7bb3a619-1dec-4710-8811-0d9709ed1d4b)

~~I would still do the other color changes, I just didn't apply them for this screenshot.~~
Edit: I updated the screenshots to merge the changes

**Comment by krestenlaust** [comment]
That is definitely better than before, it's much more clear where the bounding boxes are. This might be a viable change now.

Should still be debated though, in general, a lot of people probably have suggestions for the websites appearance. This is one of those topics where you'd need someone to call the shots to make sure the experience is streamlined

**Comment by palitu** [comment]
It looks like relatively easy changes and I think that it looks better than before.  the green selector buttons irk me!

I think this, even if not final has more appeal.

**Comment by pkrasicki** [comment]
I made a few small improvements and took some more screenshots:
![main-page2](https://github.com/LemmyNet/lemmy-ui/assets/49721209/068abe2b-592f-48ae-87cd-e0128f643e33)

![post1](https://github.com/LemmyNet/lemmy-ui/assets/49721209/93e434cf-0ef2-463c-8c45-1d5447949dda)

![post2](https://github.com/LemmyNet/lemmy-ui/assets/49721209/202eb3c1-7e64-4c87-8b58-75549f644dbd)

![create-post](https://github.com/LemmyNet/lemmy-ui/assets/49721209/108c4373-cd52-488c-a836-e928269c5a14)

Dark theme:
![dark-theme](https://github.com/LemmyNet/lemmy-ui/assets/49721209/7b0d35f9-0c0f-42a5-8690-4920007425f2)

I made a Firefox addon, which will apply those styles for you when browsing Lemmy:
https://addons.mozilla.org/firefox/addon/lemmy-modern-ui-theme
https://github.com/pkrasicki/lemmy-modern-ui-theme-addon

**Comment by krestenlaust** [comment]
It looks great now, I think you should go for it with a PR, can't imagine they'd turn it down, with the effort you've shown :)

**Comment by dullbananas** [comment]
The white boxes and the navigation bar should have the same shadow style

**Comment by rstanuwijaya** [comment]
Looks nice! I like the contrast between the cards and the page background

**Comment by krestenlaust** [comment]
Maybe in another issue, we could consider trying to implement 60-30-10 or a similar color rule to have a neat color palette

**Comment by pkrasicki** [comment]
> The white boxes and the navigation bar should have the same shadow style

![main-page3](https://github.com/LemmyNet/lemmy-ui/assets/49721209/ea2af6fe-959b-416d-827b-cfa0b79b3c41)

![main-page3-dark](https://github.com/LemmyNet/lemmy-ui/assets/49721209/b4e5a083-0b52-4cac-8203-a3f40f3c0fc9)
